_includes/v22.2/backups/backup-options.md

```diff
@@ -1,7 +1,7 @@
+`revision_history`<a name="with-revision-history"></a>           | [`BOOL`](bool.html) / None                     | Create a backup with full [revision history](take-backups-with-revision-history-and-restore-from-a-point-in-time.html), which records every change made to the cluster within the garbage collection period leading up to and including the given timestamp.<br><br>You can specify a backup with revision history  without any value e.g., `WITH revision_history`. Or, you can explicitly define `WITH revision_history = 'true' / 'false'` — this is useful for setting and unsetting this option when {% if page.name == "alter-backup-schedule.md" %} [applying different options to scheduled backups](#apply-different-options-to-scheduled-backups). {% else %} [altering a backup schedule](alter-backup-schedule.html). {% endif %} 
 `encryption_passphrase`<a name="with-encryption-passphrase"></a> | [`STRING`](string.html) | The passphrase used to [encrypt the files](take-and-restore-encrypted-backups.html) (`BACKUP` manifest and data files) that the `BACKUP` statement generates. This same passphrase is needed to decrypt the file when it is used to [restore](take-and-restore-encrypted-backups.html) and to list the contents of the backup when using [`SHOW BACKUP`](show-backup.html). There is no practical limit on the length of the passphrase.
-`DETACHED`<a name="detached"></a>                           | N/A                     |  When a backup runs in `DETACHED` mode, it will execute asynchronously. The job ID will be returned after the backup [job creation](backup-architecture.html#job-creation-phase) completes. Note that with `DETACHED` specified, further job information and the job completion status will not be returned. For more on the differences between the returned job data, see the [example](backup.html#run-a-backup-asynchronously) below. To check on the job status, use the [`SHOW JOBS`](show-jobs.html) statement. <br><br>To run a backup within a [transaction](transactions.html), use the `DETACHED` option.
+`detached`<a name="detached"></a>                           | [`BOOL`](bool.html) / None                   |  When a backup runs in `detached` mode, it will execute asynchronously. The job ID will be returned after the backup [job creation](backup-architecture.html#job-creation-phase) completes. Note that with `detached` specified, further job information and the job completion status will not be returned. For more on the differences between the returned job data, see the [example](backup.html#run-a-backup-asynchronously). To check on the job status, use the [`SHOW JOBS`](show-jobs.html) statement. Backups running on a schedule have the `detached` option applied implicitly.<br><br>You can specify a `detached` backup without any value e.g., `WITH detached`, or you can explicitly define `WITH detached = 'true' / 'false'`.<br><br>To run a backup within a [transaction](transactions.html), use the `detached` option.
```

Comment:
ALTER shouldn't accept this parameter, as a schedule requires detached to be true.

Reply:
@benbardin OK, so do we want to mention the possibility of setting the detached value true/false? A user would only be able to use this with `BACKUP`, right? If so, what scenario are we be documenting this for?

Reply:
Yeah - we thought we were being clever, but maybe we were not :/ That's exactly right - `DETACHED` is available only for CREATE/ALTER BACKUP. It is syntactically correct for ALTER BACKUP SCHEDULE as well - we wanted to reuse the component to enforce parity - but will be programatically rejected for ALTER BACKUP SCHEDULE. Does that make sense?

Reply:
I have removed the true/false for detached and made some adjustments so that the options on the two schedule pages (create/alter) are different to the description on the `BACKUP` page. Let me know what you think!